static/chats.js

```diff
@@ -59,10 +59,15 @@ class Zinja {
   }
 
   formateRindu() {
```

Comment:
Piedāvāju veikt nelielus uzlabojumus, citādi šī metode paliek sarežģīta.

Man viss sanāk šādi:
```js
  formateRindu() {
    const aktivs = this.vards == vards ? 'user' : ''
    const laiks = this.laiks ? this.laiks : '-'
    return (
      `<li class='left ${aktivs} clearfix'>
        <div class='chat-body ${aktivs} clearfix'>
          ${this.vards}: ${this.zinja}, nosūtīts: ${laiks}
        </div>
      </li>`
    )
  }
```

Bet, tad jāpielabo šī funkcija:
```js
  raadiChataRindas() {
    let chatUL = document.getElementById("chats");
    // novaacam ieprieksheejo saturu
    while (chatUL.firstChild) {
        chatUL.firstChild.remove();
    }
    // pievienojam visas zinjas
    for (const zinja of this.zinjas) {
      chatUL.innerHTML += zinja.formateRindu()
    }
    // noskrolleejam uz leju pie peedeejaa chata texta
    var chatScrollBox = chatUL.parentNode;
    chatScrollBox.scrollTop = chatScrollBox.scrollHeight;
  }```

Bet neliela piebilde - varētu kaut kādu konkrētāku klases nosaukumu, piemēram: "user-is-message-owner", vai kaut ko tamlīdzīgu.

Domāju kāpēc abās vietās jāliek ir klase, bet tur tas rombs dzīvi bojā.